src/collections/_documentation/development/sdk-dev/event-payloads/threads.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Threads Interface
+sidebar_order: 10
+---
+
+The threads interface allows you to specify the threads there were running at
+the time an event happened. These threads can also contain stack traces. As per
+policy the thread that actually crashed with an exception should not have a
```

Comment:
```suggestion
policy, the thread that crashed with an exception should not have a
```
What "policy?" Is this a Sentry specific policy or the policy of something else? I'd recommend being specific about the policy. For example, "As per Sentry programming policy..."